Touch Bar Simulator/AppDelegate.swift

```diff
@@ -53,9 +53,9 @@ extension AppDelegate: NSMenuDelegate {
 
 		menu.addItem(NSMenuItem(title: "Docking", action: nil, keyEquivalent: ""))
 		var statusMenuDockingItems: [NSMenuItem] = []
-		statusMenuDockingItems.append(NSMenuItem("Floating").bindChoice(to: .windowDocking, value: .floating))
-		statusMenuDockingItems.append(NSMenuItem("Docked to Top").bindChoice(to: .windowDocking, value: .dockedToTop))
-		statusMenuDockingItems.append(NSMenuItem("Docked to Bottom").bindChoice(to: .windowDocking, value: .dockedToBottom))
+		statusMenuDockingItems.append(NSMenuItem("Floating").streamChoice(to: .windowDocking, value: .floating))
+		statusMenuDockingItems.append(NSMenuItem("Docked to Top").streamChoice(to: .windowDocking, value: .dockedToTop))
+		statusMenuDockingItems.append(NSMenuItem("Docked to Bottom").streamChoice(to: .windowDocking, value: .dockedToBottom))
```

Comment:
Why the `stream` naming? `bind` is already a common term for binding values in Cocoa. I would also name `bindChoice` => `bindChecked` as that's more explicit about what it's doing.

Reply:
I decided it would be a good idea to avoid the `bind` naming because it has the implication that the relationship is automatically two-way, i.e., a change in the view populates to the model and a change in the model populates to the view. These methods currently just set the given value on the view _once_, when called, and perform the normal "bind" activities going from view → model (`defaults`). Thus, `stream`. Although, to be more generally useful, they should totally just observe `defaults` and do the `bind` stuff :), and that's now done.